Sources/Shared/Models/PublishViewModels.swift

```diff
@@ -0,0 +1,175 @@
+//
+//  PublishViewModels.swift
+//  VimeoNetworking
+//
+//  Copyright © 2019 Vimeo. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+/// A structure representing a single post to Facebook.
+public struct PublishToFacebookPost {
+
+    /// The title of the post as it will appear on Facebook.
+    public let title: String
+
+    /// The description of the post as it will appear on Facebook.
+    public let description: String
+
+    /// The identifier of the page being posted to on Facebook.
+    public let pageID: Int
```

Comment:
instead of `pageID: Int` must be `destination: Int`
https://github.vimeows.com/Vimeo-API/design/pull/148/files#diff-1f1bcce0884b841f7fd76d22ef138caeR27